docs/using-hive.md

```diff
@@ -350,6 +376,27 @@ and replace the contents of `platform` with:
     region: us-east1
 ```
 
+For IBM Cloud, replace the contents of `compute.platform` and `controlPlane.platform` with:
```

Comment:
```suggestion
For IBM Cloud, replace the contents of `compute.platform` and `controlPlane.platform` with your desired instance type:
```

Reply:
I tacked this onto the end:

Note that `type` is any valid IBM Cloud instance type. `type` may be omitted to use OpenShift installation defaults.